composer/algorithms/alibi/alibi.py

```diff
@@ -98,26 +103,29 @@ class Alibi(Algorithm):
     This algorithm modifies the model and runs on Event.INIT. This algorithm
     should be applied before the model has been moved to accelerators.
 
+    Example: Waiting on language model test fixtures
+
     Args:
-        heads_per_layer: number of attention heads per layer
-        max_sequence_length: maximum sequence length that the
+        heads_per_layer (int): number of attention heads per layer
+        max_sequence_length (int): maximum sequence length that the
             model will be able to accept without returning an error
```

Comment:
error, like in OOM?

Reply:
More tokens than the attention matrix is wide; the input has more elements than the module can operate on. I should just remove the "error" language. I think it's safe to assume anyone doing NLP will be familiar with it.